src/AppBundle/Resources/translations/messages.fr.yml

```diff
@@ -113,3 +113,16 @@ search.help: >
 meta.title: Plateforme de livraison de repas open source. Seulement pour les sociétés coopératives.
 
 About: À propos
+Open: Ouvert
+Closed: Fermé
+Contact us: Contactez nous
+Terms of service: Conditions d'utilisation
+Continue to payment: Paiement
+You don't have any restaurants yet: Vous n'avez pas encore de restaurants
+Create restaurant: Créer un restaurant
+Upload an image: Télécharger une image
+Default: Défaut 
+Serves cuisine: Cuisines proposées
```

Comment:
I think `Type de plats` is better. (`Cuisine` stands for meal types as Burger, Italian, Greek, aso)